linera-chain/src/manager.rs

```diff
@@ -325,9 +380,13 @@ impl MultiOwnerManager {
     }
 
     /// Resets the timer if `round` has just ended.
-    fn update_timeout(&mut self, round: RoundNumber, now: Timestamp) {
+    fn update_timeout(&mut self, round: Round, now: Timestamp) {
         if self.current_round() <= round {
-            let factor = round.0.saturating_add(2);
+            let factor = if let Round::SingleLeader(r) = round {
+                r.saturating_add(2)
```

Comment:
Why 2 and not 1?
How do you feel about a quadratic timeout?

Reply:
At the beginning of `SingleLeader(0)`, the factor is 1. At the _end_ of `SingleLeader(0)`, i.e. the beginning of `SingleLeader(1)`, it should already be 2.

Yes, quadratic makes sense in theory. But in practice long timeouts can be pretty bad. I'd like to give more options for this in a separate PR. E.g. maybe you could configure a number `N`, so that for `N` rounds in a row the factor stays the same, and only after round kN it increases to k².